samples/nrf9160/modem_shell/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -75,5 +62,5 @@ CONFIG_NET_TC_TX_COUNT=1
 CONFIG_NET_MAX_CONTEXTS=5
 CONFIG_NET_MGMT_EVENT_STACK_SIZE=1024
 
-# Needed to handle more scan results
-CONFIG_HEAP_MEM_POOL_SIZE=100000
+# Wi-Fi shell sample has 25000 heap size. We need a bit more for location request
```

Comment:
I'd like to see similar comment here than is in PR describtion. I.e. to remind when we need to increase CONFIG_NRF700X_SCAN_LIMIT and thus also heap.

Reply:
Please check the comments.